api/rest/swagger.yaml

```diff
@@ -0,0 +1,796 @@
+
+    post:
+      summary: Add a new Peer to the Cluster
+      tags:
+        - peers
+      description: Add a new Peer to the Cluster
```

Comment:
The description should be a description not a copy of the summary.

That said, remove this from the docs altogether. This should not be used.